broker/lib/controllers/ServiceFabrikApiController.js

```diff
@@ -465,13 +465,10 @@ class ServiceFabrikApiController extends FabrikBaseController {
           operationType: CONST.APISERVER.RESOURCE_TYPES.DEFAULT_BACKUP,
           operationId: backupGuid,
         }))
-      .then(result => this.backupStore.getBackupFile(result)
```

Comment:
why is this deleted ?

Reply:
its added https://github.com/cloudfoundry-incubator/service-fabrik-broker/pull/325/files/cfc8e173ee5f78f30bd39a4c6130a2084f785401#diff-b35054ba7edff4ba9736b79267cb63a2R278

Reply:
yes , spoke to @subhankarc and understood the change.